src/contracts/CowProtocolVirtualToken.sol

```diff
@@ -7,19 +7,19 @@ import "./mixins/Claiming.sol";
 import "./mixins/MerkleDistributor.sol";
 import "./vendored/mixins/StorageAccessible.sol";
 
-/// @dev The token that manages how the CowSwap governance token is distributed
-/// to all different types of investors.
-/// @title CowSwap Virtual Token
-/// @author Gnosis Developers
-contract CowSwapVirtualToken is
+/// @dev The token that manages how the CoW Protocol governance token is
+/// distributed to all different types of investors.
+/// @title CoW Protocol Virtual Token
+/// @author CoW Protocol Developers
+contract CowProtocolVirtualToken is
     NonTransferrableErc20,
     Vesting,
     Claiming,
     MerkleDistributor,
     StorageAccessible
 {
     string private constant ERC20_SYMBOL = "vCOW";
-    string private constant ERC20_NAME = "Virtual CowSwap Token";
+    string private constant ERC20_NAME = "Virtual CoW Protocol Token";
```

Comment:
Should this match the contract name as well?
```suggestion
    string private constant ERC20_NAME = "CoW Protocol Virtual Token";
```

Reply:
Fixed, now the order of the attributes is the same.